PatternFinding.py

```diff
@@ -81,25 +83,29 @@ def FindingQRPatterns(self, nooflevels):
                 x, y, w, h = cv.boundingRect(self.Contours[mapping])
                 cv.rectangle(
                     self.image, (x, y), (x + w, y + h), (0, 255, 0), 2)
-                cv.imshow('hello', self.image)
+                #cv.imshow('contour of qr', self.image)         #uncomment to debug
+                #cv.waitKey(0)
                 QRPatterns.append(patterns[ind])
+            #cv.destroyAllWindows()
         if len(QRPatterns) > 3:
             QRPatterns_new = []
             area_patterns = np.array([
                 cv.contourArea(QRpattern) for QRpattern in QRPatterns])
             arg_areapatterns = np.argsort(area_patterns)
-            for i in xrange(3):
+            for i in xrange(3):                     #pick the best three
                 QRPatterns_new.append(QRPatterns[arg_areapatterns[
                     len(arg_areapatterns) - i - 1]])
                 x, y, w, h = cv.boundingRect(QRPatterns_new[i])
                 cv.rectangle(
                     self.image, (x, y), (x + w, y + h), (0, 0, 255), 2)
-                cv.imshow('hello', self.image)
+                #cv.imshow('best qr contour', self.image)           #uncomment to debug
+                #cv.waitKey(0)
+            #cv.destroyAllWindows()
```

Comment:
indentation?

Reply:
I didn't get it.

Reply:
cv. destroyAllWinddows doesn't seem to be properly indented